WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -474,6 +476,18 @@ extension PointOfSaleAggregateModel {
     }
 }
 
+// MARK: - Lifecycle
+@available(iOS 17.0, *)
+extension PointOfSaleAggregateModel {
+    func exitPointOfSaleTapped() {
+        // Before exiting point of sale, we warn the merchant about losing their in-progress order.
+        // We need to clear it down as any accidental retention can cause issues especially when reconnecting card readers.
+        orderController.clearOrder()
+        startPaymentOnCardReaderConnection?.cancel()
+        cardReaderDisconnection?.cancel()
```

Comment:
Do you see value in canceling all the other `cancellables`?

Reply:
I can't see a downside, and it does seem strange to just cancel some of them. Updated in 0b18d06